vignettes/extending-riskmetric.Rmd

```diff
@@ -53,19 +53,19 @@ which other assessments can refer to or build off of.
 Importantly, not all references to packages are equal. We can collect
 information given the source code, a locally installed package or by scraping
 data about a package from the web. There's a hierarchy of subclasses that
-encapsulate these disperate use cases. 
+encapsulate these disparate use cases.
 
 ```
 pkg_ref
  ├─ pkg_source
  ├─ pkg_install
  └─ pkg_remote
      ├─ pkg_cran_remote
-     └─ pkg_bioc_remote
+     └─ pkg_bioc_remoteimplementations
```

Comment:
This looks like it was accidentally typed into the wrong place when trying to fix the typo below. 

```suggestion
     └─ pkg_bioc_remote
```